src/locales/en/translation.json

```
@@ -223,6 +223,8 @@
     "validateURL": "Please validate provided URL",
     "validateURLOrGoToFirstPage": "Please check the URL or <FirstPageLink />.",
     "invalidUrl": "Invalid URL",
+    "reverted": "Reverted:",
```

Error messages are usually all lower-case:
```suggestion
    "reverted": "reverted:",
```